### ISSUE TITLE
minimum_trials breaks if argument is larger than certain number

### ISSUE THREAD

**Issue by musslick** [sweetpea-org/sweetpea-py#41]
In this minimum example, the SweetPea finds a valid experiment sequence:

```
from sweetpea.primitives import factor, derived_level, within_trial
from sweetpea.constraints import minimum_trials, exclude
from sweetpea import fully_cross_block, synthesize_trials_non_uniform, \
    print_experiments, tabulate_experiments, experiment_to_csv

# color and word factors

color      = factor("color",  ["red", "green"])

# constraints

trial_constraint = minimum_trials(4)

constraints = [trial_constraint]

# experiment

design       = [color]
crossing     = [color]
block        = fully_cross_block(design, crossing, constraints,
                                 require_complete_crossing=False)

experiments  = synthesize_trials_non_uniform(block, 1)

print_experiments(block, experiments)
```

However, if we change the minimum_trials constraint to:

`trial_constraint = minimum_trials(5)`

SweetPea no longer finds a valid experiment sequence.

**Comment by mflatt** [comment]
I'm working on this problem. Recent changes turn out to address the original problem with this example, but the example still doesn't work, because it now exposes new problems with handling mininum_trials in the combinatoric sampler.

**Comment by mflatt** [comment]
Remaining problems fixed in cb9b190599f6c3164fdf55dc7211ee5b42f5dccf.
